pkg/cmd/cli/cmd/edit.go

```diff
@@ -59,14 +71,20 @@ saved copy to include the latest resource version.`
 
 // NewCmdEdit implements the OpenShift cli edit command
 func NewCmdEdit(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
-	filenames := []string{}
+	var options EditOptions
 	cmd := &cobra.Command{
 		Use:     "edit (RESOURCE/NAME | -f FILENAME)",
 		Short:   "Edit a resource on the server",
 		Long:    editLong,
 		Example: fmt.Sprintf(editExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
-			err := RunEdit(fullName, f, out, cmd, args, filenames)
+			if err := options.Complete(f, args, cmd, out); err != nil {
+				cmdutil.CheckErr(err)
+			}
+			/*if err := options.Validate(args); err != nil {
```

Comment:
Uncomment this; also `Validate` and `RunEdit` shouldn't have any parameters - everything should be used from `EditOptions`.